EngineeringUnits/BaseUnit.cs

```diff
@@ -100,9 +100,17 @@ public void Transform(UnknownUnit a)
         }
 
         public static UnknownUnit operator *(BaseUnit a, BaseUnit b) => BaseUnit.DoMath(a, b, MathEnum.Multiply);
```

Comment:
Consider that the product of a `Frequency` and a `Duration` is also dimensionless.